.github/workflows/release.yml

```
@@ -10,20 +10,88 @@ on:
+  # The release PR removes individual changesets to prepare for a release.
+  # This means that once the release PR is merged, there are no changesets left.
+  # When there are no changesets left, we can run the cli-install-cross-platform-release-test
+  # workflow to verify that the CLI installs correctly on all platforms.
+  # In all other cases, we already have a barebones `cli-install` test on the default CI platform
+  # which will catch most issues before any offending PR is merged.
+  cli-install-cross-platform-release-test:
+    needs: prepare-release
+    if: needs.prepare-release.outputs.hasChangesets == 'false'
+    strategy:
+      matrix:
+        os: [depot-ubuntu-latest, depot-macos-latest, depot-windows-2022]
```

actionlint (1.7.7)

label "depot-ubuntu-latest" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2022", "windows-2019", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "ubuntu-20.04", "macos-latest", "macos-latest-xl", "macos-latest-xlarge", "macos-latest-large", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xl", "macos-14-xlarge", "macos-14-large", "macos-14", "macos-13-xl", "macos-13-xlarge", "macos-13-large", "macos-13", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)